src/components/TodosComponents/TodosInfo.tsx

```diff
@@ -0,0 +1,161 @@
+import classNames from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo,
+  isAdding?: boolean,
+  deletingTodo?: (id: number) => void,
+  idsForLoader: number[],
+  changeTodo: (id: number, changes: Partial<Todo>) => void,
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todos,
+  isAdding,
+  deletingTodo,
+  idsForLoader,
+  changeTodo,
+}) => {
+  const { title, id, completed } = todos;
+
+  const [isDeleting, setIsDeleting] = useState(false);
+  const [isFormExist, setIsFormExist] = useState(false);
+  const [newTitle, setNewTitle] = useState(title);
+
+  const TodoField = useRef<HTMLInputElement>(null);
+  const LoaderCondition = isAdding || isDeleting || idsForLoader.includes(id);
```

Comment:
these are variables, not classes
```suggestion
  const todoField = useRef<HTMLInputElement>(null);
  const loaderCondition = isAdding || isDeleting || idsForLoader.includes(id);
```